storybook/manager.tsx

```diff
@@ -0,0 +1,37 @@
+// This file is used by storybook to register installed addons.
+// See: https://storybook.js.org/docs/addons/using-addons/
+import React from "react"
+import { addons } from "@storybook/addons"
+import { defaultTheme } from "@kaizen/design-tokens"
+import { CATEGORIES_ICON } from "./constants"
+import KaizenTheme from "./theme"
+
+const colors = defaultTheme.color
+
+addons.setConfig({
+  theme: KaizenTheme,
+  analyticsGTM: {
+    gtmId: "GTM-KS4VWLT",
+  },
+  sidebar: {
+    renderLabel: item =>
+      item.type === "root" ? (
+        <span
+          style={{
+            color: colors.purple["800"],
```

Comment:
Would be awesome to loop back an maybe tail-windify this as some point but rather than sink any more time into it lets forge ahead with this :shipit: